src/debugging/TNSDebugging.h

```diff
@@ -257,9 +256,11 @@ static void TNSEnableRemoteInspector(int argc, char **argv) {
                            dispatch_get_main_queue(), ^(int token) {
       isWaitingForDebugger = YES;
       NSLog(@"NativeScript waiting for debugger.");
-      CFRunLoopPerformBlock(CFRunLoopGetCurrent(), kCFRunLoopDefaultMode, ^{
-          CFRunLoopRunInMode(kCFRunLoopDefaultMode, 30, false);
-      });
+      NSRunLoop *currentRunLoop = [NSRunLoop currentRunLoop];
+
+      do {
+        [currentRunLoop runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
```

Comment:
Is it possible to use something like `CFRunLoopRunInMode(kCFRunLoopDefaultMode, 0.1, false)`?